actions/generate_individual_changelog.go

```diff
@@ -0,0 +1,42 @@
+package actions
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"os"
+
+	"github.com/codegangsta/cli"
+	"github.com/deis/deisrel/changelog"
+	"github.com/google/go-github/github"
+)
+
+// GenerateIndividualChangelog is the CLI action for creating a changelog for a single repo
+func GenerateIndividualChangelog(client *github.Client, dest io.Writer) func(*cli.Context) error {
+	return func(c *cli.Context) error {
+		repoName := c.Args().Get(0)
+		sha := c.Args().Get(2)
+		vals := &changelog.Values{
+			OldRelease: c.Args().Get(1),
+			NewRelease: c.Args().Get(3),
+		}
+		if vals.OldRelease == "" || vals.NewRelease == "" || sha == "" {
```

Comment:
fixed in https://github.com/deis/deisrel/pull/47/commits/2f261b319603214e2ec4a06ae16844439b7e7b42